uvm/impl/com/untangle/uvm/NotificationManagerImpl.java

```diff
@@ -343,7 +343,7 @@ private void testDiskFree(List<String> notificationList)
             File rootFile = new File("/");
             long totalSpace = rootFile.getTotalSpace();
             long usedSpace = rootFile.getUsableSpace();
-            percentUsed = (int ) ((usedSpace/totalSpace) * 100);
+            percentUsed =(int) ((1-((double) usedSpace / totalSpace) )* 100);
```

Comment:
As part of the separation of concern, couldn't we have this logic in a static method inside `SystemManagerImpl` and use that method in couple of instances ?